stvs/SageTV7/SageTV7.xml

```diff
@@ -67182,7 +67182,7 @@
           </Action>
           <Action Name="&quot;REM Option for: Configure Placeshifter Connection&quot;" Sym="OPUS4A-210962">
            <Conditional Name="!IsEmbeddedSystem()" Sym="OPUS4A-211270">
-            <Conditional Name="( IsServerUI() || (!IsDesktopUI() &amp;&amp; IsRemoteUI()) || (IsLinuxOS() &amp;&amp; GetServerProperty(&quot;linux/enable_nas&quot;,false)) ) &amp;&amp; IsSageTVServerEnabled()" Sym="BASE-45755">
+            <Conditional Name="IsLocalConnection() &amp;&amp; IsSageTVServerEnabled()" Sym="BASE-45755">
```

Comment:
The problem with this is that it will also return true for SageTVClient. And the UPnP automatic configuration won't work if run through SageTVClient on the same LAN. The reason is that in the STV code, it actually ends up getting the local IP address and then using that to forward the ports...it does this in the UI code so it can give feedback to the user about specific failures setting up UPnP.  So if you used SageTVClient, it would forward the traffic to the client and not the server...this would get fixed after a little while when the server notices the new property settings and then fixes the mapping...but would fail when the user tries to test the locator connection before that occurs. If you put it back the way it was and just change the !IsDesktupUI() part to be IsLocalConnection() then you won't have that problem.  Also, by having IsServerUI() in there, it will allow SageTVClient running on the same machine as SageTV server to configure it  (and I just noticed it would also allow placeshifter running on the same machine as the server to configure it as well).